Makefile

```diff
@@ -15,17 +15,18 @@
 #
 
 MAJOR := 0
-MINOR := 14
-PATCH := 2
+MINOR := 15
+PATCH := 0
 LABEL :=
 VERSION := $(MAJOR).$(MINOR).$(PATCH)$(LABEL)
 
 TOOLS_DEB    := machine-emulator-tools-v$(VERSION).deb
 TOOLS_IMAGE  := cartesi/machine-emulator-tools:$(VERSION)
 TOOLS_ROOTFS := rootfs-tools-v$(VERSION).ext2
+TOOLS_LIBCMT := libcmt-v$(VERSION).deb
```

Comment:
Ideally we should rename this package to `libcmt-dev` and also start providing `libcmt`.
Where `libcmt-dev` contains headers, static library and pkgconfig, but `libcmt` contains just `.so` files and nothing more.

Maybe don't need to create `libcmt` now, but I think we should at least rename it to `-dev` because the name would become repurposed and cause confusion.

`libcmt.so` will be definitely needed for people wanting to use libcmt directly from interpreted languages with C FFI like Python and LuaJIT.